rdkit/Chem/Descriptors.py

```diff
@@ -17,15 +16,14 @@ def _isCallable(thing):
   return (hasattr(collections,'Callable') and isinstance(thing,collections.Callable)) or \
               hasattr(thing,'__call__')
 
-
 _descList = []
 
 
 def _setupDescriptors(namespace):
   global _descList, descList
-  from rdkit.Chem import GraphDescriptors, MolSurf, Lipinski, Fragments, Crippen, Descriptors3D
+  from rdkit.Chem import GraphDescriptors, MolSurf, Lipinski, Fragments, Crippen
   from rdkit.Chem.EState import EState_VSA
-  mods = [GraphDescriptors, MolSurf, EState_VSA, Lipinski, Crippen, Descriptors3D, Fragments]
```

Comment:
there is one: https://github.com/rdkit/rdkit/blob/master/rdkit/Chem/Descriptors3D.py